src/components/Community/Cards.tsx

```diff
@@ -2,25 +2,29 @@ import React, { FC } from 'react';
 import { Theme } from '../../context/ThemeProvider';
 import Avatar from '../core/Avatar';
 import Card from '../core/Card';
-import Typography from '../core/Typography';
+import { UserModel } from '../../models/User';
 
 interface Tag {
   text: string;
 }
 
-export interface ProjectCardProps {
-  name: string;
-  job: string;
-  company: string;
-  tag?: string;
+export interface ProjectCardProps extends UserModel {
+  __typename: string;
+  memberof: {
+    groups: Array<{ name: string }>;
+  };
 }
 
-export const PeopleCard: FC<ProjectCardProps> = ({ name, job, company, tag }) => {
-  const tagProps = tag
-    ? {
-        text: tag || '',
-      }
-    : undefined;
+export const PeopleCard: FC<ProjectCardProps> = ({ name, __typename, ...data }) => {
+  const groups = data.memberof?.groups.map(g => g.name);
+
+  const tagProps = () => {
+    if (__typename === 'UserGroup') return { text: 'Group' };
+    if (groups?.includes('global-admins') || groups?.includes('ecoverse-admins')) {
```

Comment:
there is are also `community-admins`

Reply:
its just a test. the real tag will be done in following task